src/karmen_backend/server/tasks/get_versions_list.py

```diff
@@ -1,24 +1,31 @@
 from server import app, celery
 import requests
+import re
 from server.database import props_storage
 
 
 @celery.task(name="get_versions_list")
 def get_versions_list():
-    UPDATES_URL = "https://karmen-updates.f1.f-app.it/pill/updates/updates.txt"
+    UPDATES_URL = "https://karmen-updates.f1.f-app.it/pill/updates/updates-list.txt"
     # not expected to change, as this will have to stay to support older clients
-    print(props_storage.get_props("versions"))
     r = requests.get(UPDATES_URL)
-    print("we just got request", r, r.status_code, r.text)
     if r and r.status_code == 200:
         versions = []
-        print("here we should get some reply tet which is", r.text)
         for line in r.text.split("\n"):
+            line = line.strip()
+            if not line:
+                continue
             if line.startswith("#"):
                 continue
-            version = line.split("\t")[0]
-            if version:
-                versions.append(version)
+            if line.count("::") != 2:
```

Comment:
Asi věc názoru, ale já bych tohle nedělal. ať to raději spadne, pokud je soubor špatně formátovaný

Takhle se nedozvíme, pokud omylem namísto seznamu updatů bude ze serveru padat např. nějaký html soubor.